Gems/Compression/Code/Include/Compression/CompressionInterfaceStructs.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/containers/span.h>
```

Comment:
These includes don't appear to be necessary to this file.  Are they here for expected future changes?

Reply:
Actually there was more code in this file for the Archive API, but I removed them, but not all the includes.
I'll fix it.